esx_service/volume_kv.py

```diff
@@ -82,7 +84,7 @@
 ACCESS_TYPES = [ACCESS_READWRITE, ACCESS_READONLY]
 
 # Filesystem type
-# This option is handled in the volume-plugin at the docker host, and tracked in volume metadata. 
+# This option is handled in the volume-plugin at the docker host, and tracked in volume metadata.
```

Comment:
And IMO this is good :-) , it automatically cleans stale trailing whitespaces which need to be cleaned on expense of a bit of extra colors in reviews.